tests/integration/installation_and_upgrade/test_update_with_yum.py

```diff
@@ -13,6 +13,8 @@ class TestYumUpdate(TestInstallationAndUpgrade):
     def test_yum_update(self):
         """ Test for lustre kernel is set to boot after yum update"""
 
+        from remote_pdb import RemotePdb
```

Comment:
Is this here on purpose?

Reply:
Yes, I wanted to try and hit this block on the builder.